projects/Mallard/src/apollo.ts

```diff
@@ -0,0 +1,22 @@
+import { ApolloClient } from 'apollo-client'
+import { InMemoryCache } from 'apollo-cache-inmemory'
+import { HttpLink } from 'apollo-link-http'
+import { resolveWeatherVisibility } from 'src/helpers/weather-visibility'
+
+const RESOLVERS = {
+    /** `Query` is the root object for any Apollo query(). */
+    Query: {
+        weatherVisibility: resolveWeatherVisibility,
+    },
+}
+
+export const createApolloClient = () => {
+    const link = new HttpLink({
+        uri: 'http://localhost:4000/',
```

Comment:
This could do with a comment?

Reply:
yes you're right. Apollo requires a URI, but we're not using it right now, since we only use the "local model" system.